components/forms/createMemberForm.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+
+const CreateMemberForm = () => {
+  return (
+    <div className="flex justify-center items-center min-h-screen">
+      <form className="max-w-sm w-full ">
+        <div className="mb-5">
```

Comment:
Are we planning to ask the user for their name every time they create a new card?

Reply:
I don't know what happened, but those were not the lines I selected lol

Reply:
you're right, maybe a dropdown will be a better option.  I tried it out and will submit with this PR

Reply:
Okayyyy